### ISSUE TITLE
[GS/CS] Large map and detail textures

### ISSUE THREAD

**Issue by ChakkiSkrip** [ValveSoftware/halflife#2177]
Is it possible to increase the maximum size of the map in steam CS 1.6? Also, I would like to see the r_detailtextures function 1 by standard, instead of 0.
https://www.youtube.com/watch?v=-TQeLV4gucE

**Comment by LucesEnBabel** [comment]
I disagree that should be enabled by default, however, the "Video" option for Detailed Textures should be enabled for all mods.

Currently only Counter-Strike: Condition Zero has this option, which can be enabled by inserting `detailed_textures "1"` in the mod's liblist.gam.

**Comment by eat1k** [comment]
> I disagree that should be enabled by default, however, the "Video" option for Detailed Textures should be enabled for all mods.
> 
> Currently only Counter-Strike: Condition Zero has this option, which can be enabled by inserting `detailed_textures "1"` in the mod's liblist.gam.

Any reasons why you disagree? About the issue, I fully agree, it would be very nice to increase the limit of the map.

**Comment by ChakkiSkrip** [comment]
> I disagree that should be enabled by default, however, the "Video" option for Detailed Textures should be enabled for all mods.
> 
> Currently only Counter-Strike: Condition Zero has this option, which can be enabled by inserting `detailed_textures "1"` in the mod's liblist.gam.

You don't agree with the detailed textures, but why? If there is an opportunity to improve the visual component for all players, why not use it? I know that you can enable this option yourself. But my goal is to set this parameter to 1 for all players, because it is rarely possible to find it in someone who has it enabled, all because they do not know about this parameter.

**Comment by SamVanheer** [comment]
A better question would be to ask why detail textures default to off to begin with. Whatever the reason is, that's the one you'll need to argue against.
